.vscode/settings.json

```diff
@@ -1,6 +1,39 @@
 {
-    "go.testTags": "all",
-    "gopls.env": {
-        "GOFLAGS": "-tags=all"
```

Comment:
If you remove the `gopls.env`, you'll lose intellisense for unit and acceptance tests on VSCode

Reply:
REALLY?!?! OH MY GOD, THATS AMAZING! I've been looking for how to fix that issue for AGES. It's been driving me nuts!

I've moved all the settings from the dev container and I didn't notice that this one was gopls one. 

Love it and added back under the other gopls settings. Having more than one setting of the same name just takes the latest.

Reply:
Hell, never saw someone freaking out by getting intellisense support 😁 👍